### ISSUE TITLE
Lists can be introduced with * or + or -

### ISSUE THREAD

**Issue by aquamatthias** [chenkelmann/actuarius#9]
Only the \* can be used in actuaris, but not the + and -

Works:

```
* One
* Two
* Three
```

Does Not Work:

```
+ One
+ Two
+ Three
```

Does Not Work:

```
- One
- Two
- Three
```

**Comment by chenkelmann** [comment]
Thanks, I'll fix that, too!

**Comment by chenkelmann** [comment]
Fixed with version 0.2.6-SNAPSHOT. I already published the SNAPSHOT to maven central. I will test the new version with my two blogs where I use Actuarius to get some additional tests. If those tests are successful, I will change the version to 0.2.6.
